src/include/detail/linalg/tdb_matrix_multi_range.h

```diff
@@ -109,23 +111,30 @@ class tdbBlockedMatrixMultiRange : public Matrix<T, LayoutPolicy, I> {
   tdbBlockedMatrixMultiRange(
       const tiledb::Context& ctx,
       const std::string& uri,
-      const std::vector<I>& column_indices,
       size_type dimensions,
+      QueryType query_type,
+      const std::vector<I>& column_indices,
```

Comment:
Please update the function doc to mention how the new args work.

Reply:
Thanks, done.